openshift/topics/tutorials.adoc

```diff
@@ -1443,13 +1500,14 @@ It may take several minutes for the {appserver_name} image to deploy.
 
 ==== Log in to the {appserver_name} Server Using {project_name}
 
+.Procedure
+
 . Access the {appserver_name} application server and click *Login*. You are redirected to the {project_name} login.
 . Log in using the {project_name} user created in the example. You are authenticated against the {project_name} server and returned to the {appserver_name} application server.
 
 
 [[Example-EAP-Manual]]
-=== Example Workflow: Manually Registering EAP Application in {project_name} with SAML Client
-This follows on from xref:Example-Deploying-SSO[Example Workflow: Preparing and Deploying the {project_openshift_product_name} image], in which {project_name} was deployed on OpenShift.
+=== Manually Registering EAP Application in {project_name} with SAML Client
```

Comment:
`Manually registering EAP application in ... client`? (looks in other sections the labels got changed to lowercase)